src/components/Modal/Modal.module.scss

```diff
@@ -30,37 +36,36 @@
   display: flex;
   position: relative;
   flex-direction: column;
-  padding: 0 var(--spacing-xl);
-  padding-block-end: var(--spacing-large);
-  border-radius: var(--border-radius-big);
-  box-shadow: var(--box-shadow-large);
-  background-color: var(--secondary-background-color);
-  max-height: 90vh;
-  max-width: 90vw;
+  padding: 32px;
```

Comment:
please make for it css variables and use it, or maybe it was already created in a global.scss